public/static/app/perf-bb/routes.js

```diff
@@ -0,0 +1,19 @@
+mciModule.config(function($routeProvider, $locationProvider) {
```

Comment:
Could you give a bit more context on what this is used for? Is it to redirect certain routes?